openpdf/src/main/java/com/lowagie/text/pdf/RandomAccessFileOrArray.java

```diff
@@ -50,24 +50,19 @@
 package com.lowagie.text.pdf;
 
 import com.lowagie.text.Document;
-import java.io.ByteArrayOutputStream;
-import java.io.DataInput;
-import java.io.DataInputStream;
-import java.io.EOFException;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.RandomAccessFile;
+
+import java.io.*;
```

Comment:
Do we have a style guide for the project? Are we using google's style guide for java? If so, wildcard imports are usually discouraged.

Reply:
There is no style guide as I know. This one is default from intellij. Will add this check to checkstyle and fix all wildcard imports.